src/assignments/benson/advent_2018/task3.clj

```diff
@@ -0,0 +1,79 @@
+(ns assignments.benson.advent-2018.task3)
+
+;;;; ------------------ Part 1 ------------------
+
+(def task3input (slurp "src/assignments/benson/advent_2018/task3input"))
+
+(def raw-claims (clojure.string/split-lines task3input))
+
+;; split claims into a vector and convert all strings to integers
+(def claims (vec (map #(vec 
+                        (map (fn [measurement] (Integer/parseInt measurement)) 
+                             (re-seq #"\d+" %))) 
+                      raw-claims)))
+;; claim = [ID X Y W H]
+;; ID: The number of the claim
+;; X: The number of inches between the left edge of the fabric and the left edge of the rectangle.
+;; Y: The number of inches between the top edge of the fabric and the top edge of the rectangle.
+;; W: The width of the rectangle in inches.
+;; H: The height of the rectangle in inches.
+
+;; ...........
+;; ...........
+;; ...#####...
+;; ...#####...
+;; ...#####...
+;; ...#####...
+;; ...........
+;; ...........
+;; ...........
+; [3 2 5 4]
+
+(defn parse-claim 
+  "Returns a vector of the coordinates of fabric placement based on
+   the claim"
+  [[_ x y w h]]
+  (for [width (range w)
+        height (range h)]
+    [(+ width x) (+ height y)]))
+
+(defn parse-all-claims
+  "Returns a vector of the coordinates of fabric placement of all CLAIMS"
+  [claims]
+  (reduce #(concat %1 (parse-claim %2)) [] claims))
```

Comment:
This bit feels wrong.   (reduce #(concat ..)) is a strange thing to do; it's like an n^2 type of algorithm.
I was expecting to see something like (map parse-claim claims) ?  Maybe your data representations are not quite right?  I'm not sure.